src/components/Shared/CrossingMapPage/CrossingMapSearchBar.js

```diff
@@ -91,9 +91,17 @@ class CrossingMapSearchBar extends Component {
     { suggestion, suggestionValue, suggestionIndex, sectionIndex, method },
   ) => {
     if (suggestion.__typename === 'Crossing') {
-      this.props.history.push(`/map/crossing/${suggestion.id}`);
+      if(this.props.location.pathname.includes('dashboard')) {
+        this.props.history.push(`dashboard/map/crossing/${suggestion.id}`);
```

Comment:
this place is `dashboard` and the line below has `/dashboard`

I'm assuming both work but consistency would be cool